src/BuildAF.js

```diff
@@ -0,0 +1,230 @@
+import React from "react";
+import styled from "styled-components";
+import {
+  CX_OFF_WHITE,
+  CX_FONT,
+  CX_DARK_BLUE,
+  BATMAN_GRAY
+} from "./Constants.js";
+import Card from "@material-ui/core/Card";
+import CardHeader from "@material-ui/core/CardHeader";
+import List from "@material-ui/core/List";
+import ListItem from "@material-ui/core/ListItem";
+import ListItemText from "@material-ui/core/ListItemText";
+
+//URL for the AF team main repo to keep track of
+const AFURL =
+  "https://jenkins.phx.connexta.com/service/jenkins/blue/rest/organizations/jenkins/pipelines/HAART-Jobs/pipelines/";
```

Comment:
❗️ Make http request instead to avoid CORS issues 
```suggestion
  "http://jenkins.phx.connexta.com/service/jenkins/blue/rest/organizations/jenkins/pipelines/HAART-Jobs/pipelines/";
```